docs/modules/getting_started/examples/snip_crc_preset_list.adoc

```diff
@@ -0,0 +1,4 @@
+* {ocp}
+* {okd}
+* {ushift}
+* Podman container runtime
```

Comment:
A variable might still be halepful to track and perhaps assist in the situation when we mention the support from he other presets.

Reply:
Maybe I missed the questioon. Example files are to put text block that can be different downstream. (As opposed to attributes that have only inline content.)

Reply:
since it is clearly a name I thought a variable would make sense. there is also `podman` the command. but up to you